### PR TITLE
Added: CSS for 1. collapse 2. figure caption ~3. code tab size~(deleted) 4. table top align && Fixed gitalk redirect 

### PR DESCRIPTION
**Pull request by upupming**
# 1. collapse/expand

screenshot from 2018-06-21 21-21-23

# 2. figure caption center align

screenshot from 2018-06-21 21-14-58

# 3. code tab size set to 2 spaces

I follow the [google java style](https://google.github.io/styleguide/javaguide.html) which uses 2-spaces for indentation. I think 2 spaces is a better choice, how about letting users config this in their `melody.yml`?

# 4. table top aligin

|Before|After|
|-|-|
|screenshot from 2018-06-16 02-03-26|screenshot from 2018-06-16 02-04-33|

Another problem: In this condition, the **copy icon**s are missing.

## Update
Fixed [gitalk redirrect issue](https://github.com/gitalk/gitalk/issues/125) by changing the `gitalk.js`.

|before|after|
|-|-|
|screenshot from 2018-06-21 22-00-48|screenshot from 2018-06-21 22-01-32|

### CODE REVIEW

**Comment by Molunerfinn** [comment]
I find gitalk.js & gitalk.min.js, just use the gitalk.min.js is OK. I think you should remove the gitalk.js.

**Comment by Molunerfinn** [comment]
I find gitalk.js & gitalk.min.js, just use the gitalk.min.js is OK. I think you should remove the gitalk.js.

File: source/css/_global/index.styl [hunk 1]
```diff
@@ -102,7 +121,8 @@ table
   th,
   td
     padding: 0.3rem 0.6rem
-    border: 1px solid $light-grey
+    border: 1px solid #dfe2e5
```
Comment (Molunerfinn) [inline source/css/_global/index.styl]: The color here, you'd better use the color in the `var.styl`.

**Comment by upupming** [comment]
But the file won't be loaded because we don't include it in the head. If I keep it, more changes can be done in the future if necessary. What do you think?

File: source/css/_global/index.styl (hunk 1 above)
Comment (Molunerfinn) [reply source/css/_global/index.styl]: If you think $light-grey is so light, then you can use the stylus built-in function `darken()`, such as `darken($light-grey, 10%)`

**Comment by Molunerfinn** [comment]
@upupming But the `gitalk.js` is not useful for the repo and expand the size. If you want to do more change for it, just update the `gitalk.min.js` is OK. And in the future, may be these third-party libs' url can be changed by user in the `melody.yml`, the `gitalk.js` will be no more used.

**Comment by Molunerfinn** [comment]
I don't understand the part of `code tab size set to 2 spaces`, what config should users do?

**Comment by upupming** [comment]
Such as in `melody.yml`
```yml
code-tab-size: 2
```

**Comment by Molunerfinn** [comment]
@upupming and then? What should the theme do?

**Comment by Molunerfinn** [comment]
@upupming and then? What should the theme do?

**Comment by upupming** [comment]
Maybe sometimes we'll copy long code from other places, but the code is indented with tabs. If the code copied from elsewhere mixed with our own code using spaces, it's awkward.

**Comment by Molunerfinn** [comment]
@upupming Then you need to fix it with your editor. Just like I can't help you to fix the problem in your article text.

**Comment by upupming** [comment]
@Molunerfinn I think you are totally right.

**Comment by Molunerfinn** [comment]
@upupming And thank you for your hard work!

**Pull request merged**